R/IO.R

```diff
@@ -1747,7 +1765,7 @@ write.FCS <- function(x, filename, what="numeric", delimiter = "|", endian = "bi
     
     #must clear the old $PnX before assigning the new ones since the old ones may not be dropped if fr was subsetted previously
     # because description<- only update or insert but does not delete the old
-    x@description <- orig.kw[!grepl("^\\$P[0-9]+[BERNS]$", names(orig.kw))]
+    x@description <- orig.kw[!grepl("\\$P[0-9]+[BERNS]", names(orig.kw))]
```

Comment:
This is the primarily offensive line, as in the prior form it will leave `flowCore_$PnRmax` key-value pairs in the description. This change causes those to be caught and removed along with any other `Pn` keys. Following the logic of the comment, these should be fully replaced due to potential subsetting (as in the issue).

Reply:
This line is removed in d78cba5